front/src/modules/conflict/components/ConflictsListV2.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import cx from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import type { ConflictV2 } from 'common/api/osrdEditoastApi';
+import type { TrainScheduleWithDetails } from 'modules/trainschedule/components/TimetableV2/types';
+
+import type { ConflictTableProps, ConflictWithTrainNames } from './ConflictCardV2';
+import ConflictCardV2 from './ConflictCardV2';
+
+function addTrainNamesToConflicts(
+  conflicts: ConflictV2[],
+  trainSchedulesDetails: TrainScheduleWithDetails[]
+): ConflictWithTrainNames[] {
+  const trainNameMap: { [id: number]: string } = {};
+
+  trainSchedulesDetails.forEach(({ id, trainName }) => {
+    trainNameMap[id] = trainName;
+  });
+
+  return conflicts.map((conflict) => ({
+    ...conflict,
+    train_names: conflict.train_ids.map((id) => trainNameMap[id] || ''),
+  }));
+}
+
+export default function ConflictsListV2({
+  conflicts,
+  expanded,
+  trainSchedulesDetails,
+  toggleConflictsList,
+  onClick,
+}: ConflictTableProps) {
```

Comment:
At least for react components, we try to declare them with arrow functions.
You then need to add `export default ConflictsListV2` at the bottom of the file.
```suggestion
const ConflictsListV2 = ({
  conflicts,
  expanded,
  trainSchedulesDetails,
  toggleConflictsList,
  onClick,
}: ConflictTableProps) => {
```

Reply:
done